verticapy/tests_new/conftest.py

```diff
@@ -523,3 +527,63 @@ def airline_vd_fun(schema_loader):
     airline = load_airline_passengers(schema_loader, "airline")
     yield airline
     drop(name=f"{schema_loader}.airline")
+
+
+@pytest.fixture(scope="module")
+def pop_growth_vd(schema_loader):
+    """
+    Create a dummy vDataFrame for load_pop_growth dataset
+    """
+    pop_growth = load_pop_growth(schema_loader, "pop_growth")
+    yield pop_growth
+    drop(name=f"{schema_loader}.pop_growth")
+
+
+@pytest.fixture(scope="function")
+def pop_growth_vd_fun(schema_loader):
+    """
+    Create a dummy vDataFrame for load_pop_growth dataset
+    """
+    pop_growth = load_pop_growth(schema_loader, "pop_growth")
+    yield pop_growth
+    drop(name=f"{schema_loader}.pop_growth")
+
+
+@pytest.fixture(scope="module")
+def gapminder_vd(schema_loader):
+    """
+    Create a dummy vDataFrame for load_gapminder dataset
+    """
+    gapminder = load_gapminder(schema_loader, "gapminder")
+    yield gapminder
+    drop(name=f"{schema_loader}.gapminder")
+
+
+@pytest.fixture(scope="function")
+def gapminder_vd_fun(schema_loader):
```

Comment:
Again
@vikash018 Any particular reason for this scope to be function? if it is not required, then you can make it broader. It will increase the speed.

Reply:
Some methods do inplace modification of data. If we run this method multiple times for a test function scope, then we need function scope to avoid data modification. other way would be creating copy of data.